app/views/stories/_sign_in_invitation.html.erb

```diff
@@ -1,24 +1,24 @@
-<div class="single-article single-article-small-pic feed-cta" id="in-feed-cta">
-  <div class="cta-container" id="cta-content">
-    <img class="secondary-logo" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
-         alt="Secondary <%= community_name %> logo" />
-
-    <h2>
-      <a href="/"><%= community_name %></a> is a community of <br />
-      <%= number_with_delimiter User.registered.estimated_count %> amazing <%= community_members_label %>
-    </h2>
-
-    <h3>
-      Create your profile to customize your experience and get involved.
-    </h3>
-
-    <div class="button-container">
-      <a href="/enter" class="cta cta-button" aria-label="Sign up.">
-        Join Now
-      </a>
-      <div class="feed-cta-sub">
-        <em>We require social login to prevent abuse.</em>
-      </div>
+<div class="authentication-feed__card" id="in-feed-cta">
+  <div class="authentication-feed__container">
+    <img class="authentication-feed__logo" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
+          alt="Secondary <%= community_name %> logo" />
+    <div class="authentication-feed__content">
+      <h2 class="authentication-feed__title">
+community_members_label
+                                                                %>
```

Comment:
I think a portion of the welcome message may have been lost along the way? Should it maybe look something like this instead?

```suggestion
<a href="/"><%= community_name %></a> is a community of <br />
        <%= number_with_delimiter User.registered.estimated_count %> amazing <%= community_members_label %>
```

So the card can render on the feed like this:

<img width="758" alt="Screen Shot 2020-08-13 at 11 23 12" src="https://user-images.githubusercontent.com/6045239/90166731-e90c1d80-dd57-11ea-8245-243c03db9835.png">

Reply:
Whoops! Let me look into this.